ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedModule.java

```diff
@@ -139,24 +128,30 @@ public void onHostResume() {
   }
 
   @Override
-  public void onBatchComplete() {
```

Comment:
This is a method native modules can override that gets called when batches are completed so it is fine to remove it since it will just call the one from the super class.